src/AbstractActionHandler.test.ts

```diff
@@ -245,8 +245,12 @@ describe('Action Handler', () => {
       lastIrreversibleBlockNumber: 1,
     }
 
-    const result = actionHandler.handleBlock(nextBlock, false)
-    await expect(result).rejects.toThrow(MismatchedBlockHashError)
+    const result = await actionHandler.handleBlock(nextBlock, false)
+    // roll back to last irreversible block number is 0
+    // @ts-ignore
+    expect(actionHandler.lastProcessedBlockNumber).toBe(0)
```

Comment:
Why 0? you should define or tell `last irreversible block` before the test to make this case clearer.